### DISCUSSION TITLE
Generate migrations files based on the changes you have made to your models like Django

### DISCUSSION THREAD

**Discussion by sgaseretto** [essenciary/SearchLight.jl#60] [Ideas]
Hello everyone! I wanted to ask one thing that I already asked in an issue at the searchlight's repo.
Are there any plans on adding the possibility to automatically generate migrations files from DataType Models? 
something that could generate migrations files based on the changes you have made to your models like [`makemigrations`](https://docs.djangoproject.com/en/4.0/topics/migrations/) from Django? I wanted to start developing a package to do this, but since searchlight has not been completely documented (and I'm fairly new to Julia), I desisted. 

Something like this will be extremely helpful to avoid repeating ourselves and reduce the possibility of making errors when defining and changing models.

**Comment by AbhimanyuAryan** [comment]
Hi,

This is a nice to have feature. Documenting searchlight is getting delayed for a long time. This feature is not on TODO list atm but documenting searchlight is definitely on todo 

I am working on Genie V5 docs right now which is kind of critical. Once we have the docs build process automated, I can move to SL documentation
